packages/dapp/src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import styled from "styled-components";
+import { Civil, CivilErrors } from "@joincivil/core";
+import NavBarItem from "./NavBarItem";
+
+const StyledUL = styled.ul`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-top: 0;
+  padding: 0;
+  height: 50px;
+  background-color: black;
+`;
+
+export interface NavBarState {
+  balance: string;
+}
+
+class NavBar extends React.Component<{}, NavBarState> {
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      balance: "loading....",
```

Comment:
this is fine, but also weird as hell. makes more sense to me to have a loading boolean then set the balance to be loading...

Reply:
I'll make sure to do something like that when I do the balance widget "for real"